packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -695,17 +695,8 @@ class _PopupMenuRouteLayout extends SingleChildLayoutDelegate {
     // childSize: The size of the menu, when fully open, as determined by
     // getConstraintsForChild.
 
-    final double buttonHeight = size.height - position.top - position.bottom;
     // Find the ideal vertical position.
```

Comment:
nit: probably we can remove this comment as we don't have to calculate the vertical position.

Reply:
Done!